SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -41,7 +41,10 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
                 "OnlineSubsystemUtils",
                 "Projects",
                 "ReplicationGraph",
-                "Sockets"
+                "Sockets",
+                "Slate",
+                "SlateCore",
+                "UMG"
```

Comment:
Double-checking these are all needed?

Reply:
Good call, turns out SlateCore is redundant when also including Slate (which is required)